datameta/api/openapi.yaml

```diff
@@ -94,6 +138,17 @@ components:
         - token
         - expiresAt
 
+    PasswordChange:
+      type: object
+      properties:
```

Comment:
maybe add a property `PasswordToken` and make both `PasswordToken` and `oldPassword` optional (either one or the other)

@lkuchenb

Reply:
Resolved by accepting an abitrary `passwordChangeCredential` which can be either the old password or a password reset token.